uaa/src/main/webapp/WEB-INF/spring/saml-providers.xml

```diff
@@ -144,6 +149,8 @@
         <bean id="samlWebSSOProcessingFilter" class="org.springframework.security.saml.SAMLProcessingFilter">
             <property name="authenticationManager" ref="authenticationManager" />
             <property name="authenticationSuccessHandler" ref="successRedirectHandler" />
+            <property name="authenticationFailureHandler" ref="samlLoginFailureHandler" />
+            <property name="contextProvider" ref="contextProvider" />
```

Comment:
@fhanik I can't tell if you're seriously thanking me or being sarcastic. No offense taken either way but let me know if you want me to remove it. Sorry I didn't catch that earlier. The trouble with annotations and conventions is that you have to know ahead of time what to look for.